src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml

```diff
@@ -9,10 +9,10 @@ imports:
 sylius_grid:
     templates:
         action:
-            show: "@SyliusUi/Grid/Action/show.html.twig"
-            pay: "@SyliusShop/Account/Order/Grid/Action/pay.html.twig"
+            shop_show: "@SyliusShop/Grid/Action/show.html.twig"
```

Comment:
I think it's a BC break - anyone using this action in their own grid will be affected.

I'd recommend introducing new actions with those prefixes and deprecating the current Twig templates from `@SyliusUi` - we'll probably need to bump up Twig requirements, see https://twig.symfony.com/doc/2.x/tags/deprecated.html.

Reply:
IMHO, the main place where we can consider BC break is here: https://github.com/Sylius/Sylius/pull/10331/files#diff-09570101558a8ee92c483bda1a32d2e0R54

That's true, it may affect all users, that turned off AdminBundle. That's why I have postulated moving from `AdminBundle` all configuration coupled to `UiBundle` to `UiBundle` itself.

Reply:
I've moved basic grid actions and filters to `UiBundle`, now both `AdminBundle` and `ShopBundle` are importing config from `UiBundle`. If I think correctly, there is no BC break right now.

Reply:
I think now the problem is, `SyliusUiBundle` does not depend (and should not) on `SyliusGridBundle`, so having a `sylius_grid` configuration in `UiBundle/Resources/config/app/config.yml` would not work if you use `UiBundle` as a standalone package... The question is, should it even be possible to use it without the rest of the Sylius, as it provides strictly Sylius-related UI :) But still, it is possible now, so the problem exists.

Reply:
I do not see much sense in using `UiBundle` as a standalone package, what is more, I've checked that before and `UiBundle`  is coupled with `GridBundle` by for example using [twig filters from `GridBundle`](https://github.com/Sylius/Sylius/blob/1.3/src/Sylius/Bundle/UiBundle/Resources/views/Grid/_default.html.twig#L22). So here nothing changes imo 😃

Reply:
I don't see any sense as well, I just say it's possible 💃 If the incorrect coupling is there anyway, I would be 👌 with the proposed changes, but we should also keep in mind to either couple the `UiBundle` with `GridBundle` in `composer.json` or (IMO better choice) don't allow to use it as standalone. 🖖

Reply:
For now, I've coupled `UiBundle` with `GridBundle` in `composer.json`